### PR TITLE
Skipping steps from inside a step

### PR DESCRIPTION
**Pull request by eerkunt**
This PR addresses the issue described in #295.

### CODE REVIEW

**Comment by timofurrer** [comment]
Thanks for the PR :tada: Looks good so far!
Can you add some documentation to: https://github.com/radish-bdd/radish/blob/master/docs/tutorial.rst as well? Maybe after the "Step Text Data" section?

**Comment by eerkunt** [comment]
Does this look ok ?

File: docs/tutorial.rst [hunk 1]
```diff
@@ -572,6 +572,61 @@ To access this text data you can use the ``text`` attribute on the ``step`` obje
...
+.. code:: python
+
+   # -*- coding: utf-8 -*-
+
+   from radish import given, when, then
+
+   @given("I have the following quote in target DB")
+   def have_quote_in_target_db(step):
+
+       ... code that would check the query in the DB
```
Comment (timofurrer) [inline docs/tutorial.rst]:
```suggestion
       # code that would check the query in the DB
```

File: docs/tutorial.rst [hunk 2]
```diff
@@ -572,6 +572,61 @@ To access this text data you can use the ``text`` attribute on the ``step`` obje
...
+    ...
+    Scenario: Test quote system
+      Given I have the following quote in target DB
+          """
+          To be or not to be
+          """
+      When I found 2 quotes in the DB
+      Then I delete one of them
+
+To skip the tests if `To be or not to be` quote could not be found:
```
Comment (timofurrer) [inline docs/tutorial.rst]:
```suggestion
To skip the step if `To be or not to be` quote could not be found:
```

File: docs/tutorial.rst [hunk 3]
```diff
@@ -572,6 +572,61 @@ To access this text data you can use the ``text`` attribute on the ``step`` obje
       assert len(step.context.database.quotes) == number
 
 
+Skipping a Step
+---------------
+
+In some situations it might be required to set the step state manually within a step.
```
Comment (timofurrer) [inline docs/tutorial.rst]:
```suggestion
In some situations it might be required to skip a step under certain conditions.
```

**Comment by eerkunt** [comment]
Yep, looks much better.

**Comment by timofurrer** [comment]
Alrighty!

Let's wait for the checks and get this merged! 🎉 Thanks!

**Comment by eerkunt** [comment]
Thanks! So how can I use this within my project ? Any plan to release a new version ?

**Comment by timofurrer** [comment]
I'll release in a few hours!

**Pull request merged**